### DISCUSSION TITLE
如何连接要求验证码的Cloudreve云盘？

### DISCUSSION THREAD

**Discussion by gexgd0419** [AlistGo/alist#3139] [Q&A]
Cloudreve可以要求提供验证码，这样若`captchaCode`为空就会无法登录。

目前alist还没有提供除了用户名+密码之外的Cloudreve连接方式。

我可以在网页端登录之后得到`cloudreve-session`的cookie。靠提供cookie来连接Cloudreve是否可行？

**Comment by anwen-anyi** [comment]
可以问问这个大佬，[Panici4](https://github.com/Panici4) 
大佬提的pr:https://github.com/alist-org/alist/pull/2997
